test/jasmine/tests/image_test.js

```diff
@@ -448,7 +544,7 @@ describe('image hover:', function() {
             .then(function() {_hover(255, 295);})
             .then(function() {
                 assertHoverLabelContent({
-                    nums: 'x: 31.5\ny: 35.5\nz: [128, 77, 54, 254]\nRGBA: [128, 77, 54, 1]',
+                    nums: 'x: 31.5\ny: 35.5\nz: [128, 77, 54, 255]\nRGBA: [128, 77, 54, 1]',
```

Comment:
Sorry for the unclear change.

This was needed because I changed the colormodel for mock `image_adventurer` from `rgba` to `rgba256`. This mock contained alpha values of `254` which were rendered as completely opaque in `rgba` because the max is `1`. However, in `rgba256`, `254` is not completely opaque because the max is `255`. Therefore, I replaced all the `254` by `255` in order for the baseline image to be exactly the same as before. However, I had to update this one test for the hover. I hope this makes sense.

cc @archmoj